src/App.js

```diff
@@ -2,6 +2,8 @@ import "./App.css";
 import { Footer, Navbar, Homepage } from "./Components/Allcomps";
 import { ProductsPage } from "./Components/Products/ProductsPage";
```

Comment:
Appreciation: good way of importing files , You are good to go . You can merge it